### PR TITLE
ISPN 10x Internal thread states exposition

### PR DESCRIPTION
**Pull request by diegolovison**
The commit https://github.com/infinispan/infinispan/commit/2b68c55bcb94fc42fad5e2ee3b63eef9d912dd90 removed the `delegate` field.
The goal with this PR is make RadarGun compatible with ISPN10x

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -430,10 +430,11 @@
             <module>../plugins/infinispan92</module>
             <module>../plugins/infinispan93</module>
             <module>../plugins/infinispan94</module>
+            <module>../plugins/infinispan100</module>
```
Comment (alanfx) [inline parent/pom.xml]: Not sure why you're picking `infinispan100` instead of `infinispan10`?

Comment (diegolovison) [reply parent/pom.xml]: `infinispan50` is ISPN5.0
`infinispan100` is ISPN 10.0
I followed that pattern

Comment (alanfx) [reply parent/pom.xml]: I guess for me, the plugin version never changes for a micro release, so I think `infinispan10` is better.

Comment (diegolovison) [reply parent/pom.xml]: I believe that we will have 10.1, 10.2, 10.3 in a short period of time instead of 11.0, 12.0 and 13.0

Comment (alanfx) [reply parent/pom.xml]: OK, you are completely correct on this! Sorry, I have having Monday morning brain. I'll merge this

**Pull request merged**